website/docs/r/organization_token.html.markdown

```diff
@@ -28,11 +28,34 @@ The following arguments are supported:
 * `force_regenerate` - (Optional) If set to `true`, a new token will be
   generated even if a token already exists. This will invalidate the existing
   token!
+* `expired_at` - (Optional) A date and time in which the organization token will expire. The expiration date must be passed in
+  iso8601 format. If no expiration date is supplied, the expiration date will default to null and never expire.
 
 ## Attributes Reference
 
 * `id` - The ID of the token.
 * `token` - The generated token.
+* `expired_at` - The token's expiration date. The expiration date must be passed in
+iso8601 format. If no expiration date is supplied, the expiration date will default to null and never expire.
```

Comment:
I'm so sorry I gave you the wrong feedback. I was looking at a single commit and not the entire change. I didn't see this was already added to the section above and I got them mixed up.

```suggestion
```